spec/spec_helper_acceptance.rb

```diff
@@ -40,6 +40,12 @@
       shell 'apt-get install -y ssl-cert'
     end
 
+    if fact('operatingsystem') == 'SLES'
+      #generate certs for SLES
+      shell 'zypper install -y apache2'
```

Comment:
👍  on making it isolated to test environments. I think installing/managing packages like that fall under the scope of the users' sys administration.